projects/helm/chartmuseum/patches/redis.patch

```diff
@@ -0,0 +1,77 @@
+diff --git a/cmd/chartmuseum/main.go b/cmd/chartmuseum/main.go
```

Comment:
I'm not sure where this patch came from, but should probably use `git format-patch` to create it and have a link to upstream issue/PR for it if there is something. I don't see a PR to add this feature in helm/chartmuseum

Reply:
It is the patch from harbor for this component. https://github.com/goharbor/harbor/blob/v2.4.1/make/photon/chartserver/redis.patch

Reply:
As suggested by Jackson, instead of checking out harbor source code and applying patches during the build time, it is preferred to copy those patches to the patches folder.